config/initializers/console.rb

```diff
@@ -1,3 +1,21 @@
+class ConsolePrompt
+  def self.generate_prompt(app_name: 'Apply/Manage')
```

Comment:
I shorted to "Apply/Manage". If you are strongly against let me know. Happy to change to a name where everyone feel comfortable.